packages/@aws-cdk/aws-dynamodb-global/README.md

```diff
@@ -0,0 +1,30 @@
+## @aws-cdk/aws-dynamodb-global
+
+Here is a minimal deployable Global DynamoDB tables definition:
+
+```typescript
+import globaldynamodb = require('@aws-cdk/aws-dynamodb-global');
+const CONSTRUCT_NAME = 'aws-cdk-dynamodb-global';
+const TABLE_PARTITION_KEY: Attribute = { name: 'hashKey', type: AttributeType.String };
+
+const stack = new Stack();
+const globalTables = new GlobalTable(stack, CONSTRUCT_NAME, {
+    dynamoProps: {
+        partitionKey: TABLE_PARTITION_KEY,
+        tableName: TABLE_NAME
+    },
+    regions: [ 'us-east-1', 'us-east-2', 'us-west-2' ]
+});
+```
+
+## Implementation Notes
+Global Dynamodb tables is an odd case currently.  The way this package works -
```

Comment:
Make sure you capitalize service names properly and use the prefix "Amazon" or "AWS" (in this case it's `Amazon DynamoDB`).